src/gsi_ncdiag/proc_gsi_ncdiag.py

```diff
@@ -6,11 +6,6 @@
 import time
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
-if not IODA_CONV_PATH.is_dir():
-    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
-sys.path.append(str(IODA_CONV_PATH.resolve()))
-
 import gsi_ncdiag as gsid
```

Comment:
Does this path need to be updated? to `import gsi_ncdiag.gsi_ncdiag as gsid` like the next file is?

Reply:
yes that's correct this will need to be changed